content/blog/2023-11-13-announcing-localstack-30-general-availability/index.md

```diff
@@ -0,0 +1,186 @@
+
+{{< img-simple src="localstack-ephemeral-environments.png" width=300 alt="Image of LocalStack Ephemeral environments with a LocalStack Sandbox running in the cloud">}}
+
+Check out our [documentation]() on getting started with Ephemeral Instances.
+
+### CI Analytics for LocalStack
+
+We're excited to introduce **CI Analytics**, a new feature for comprehensive state browsing and analysis of historical continuous integration (CI) builds. This feature integrates into your LocalStack CI workflow, offering insights and supporting our goal to enhance the cloud developer experience throughout the software development lifecycle (SDLC). With CI Analytics, you can collect, analyze, and visualize critical metrics from your software CI pipelines, helping you understand the impact of cloud infrastructure changes on CI builds. It facilitates root cause analysis for build failures, supports data-driven decisions for continuous improvement, and more.
+
+CI Analytics combines a number of existing features in the LocalStack platform, such as [Cloud Pods]() & [Stack Insights](), and provides a unified view of the state of your LocalStack resources across CI builds. The features include:
+
+- **CI Project Runs**: This gives you a consolidated view of all CI builds for a specific project.
+- **Log Output**: You can view detailed log output for individual CI builds.
+- **Request/Response Traces**: This feature provides traces of requests and responses, including details of the AWS service and operation, along with expandable request and response payloads.
+- **Cloud Pod State**: It allows you to view the state of LocalStack resources for a particular CI build through a Cloud Pod, which can be injected locally or loaded in an Ephemeral Instance.
+- **Stack Insights**: This offers a view and audit log of interactions, API error codes in your application stack over time, and enables you to drill down into the most commonly used services and API calls.
+
+{{< img-simple src="localstack-ci-analytics.png" width=300 alt="Image of LocalStack CI Analytics dashboard">}}
+
+Check out our [documentation]() on getting started with CI Analytics.
+
+### All-new LocalStack Desktop Application
+
+We're excited to announce the release of the **LocalStack Desktop Application**, our cross-platform Desktop client. The Desktop Application features an intuitive user interface, with local [Resource Browser](https://docs.localstack.cloud/user-guide/web-application/resource-browser/) for 30+ AWS services and a simplified way to manage LocalStack containers and logs. The new Desktop Application replaces the LocalStack Cockpit, streamlining the developer experience and making our cloud emulator tooling more accessible. [Download it for free and get started!](https://app.localstack.cloud/download).
```

Comment:
capitalization 'desktop app'

Reply:
I'd not be specific about 'for 30+ AWS services'. with that it seems to me that desktop app supports a different set of services in RB

Reply:
Its just to mention that we have these set of RBs on the Desktop app. This is one of the selling points for the app and we have used similar wordings in our Desktop app blog

Reply:
yes, don't specify the number. or something like 'RB supporting all the services like our web app' (with better wording... :D)